### PR TITLE
Avoid compression to improve BitmapWrapper performance

### PR DESCRIPTION
**Pull request by byencho**
This PR tweaks how we read / write a `Bitmap` (via `BitmapWrapper`) to prefer speed over compression. Right now we write the data to a stream using

```java
mBitmap.compress(Bitmap.CompressFormat.PNG, 100, stream);
```

and then convert that stream to a `byte` array. Instead we can use the optimized `Bitmap.copyPixelsToBuffer` call to write the data as-is.

I would not expect this to really affect any real-world usage of `Bridge` but we save / restore `Bitmap`s in the sample app and preferring speed helps with testing. I'll also note that even though the lack of compression results in a larger amount of disk space used, all of the space used by `Bridge` is temporary so this shouldn't be _too_ much of a concern.

### CODE REVIEW

File: bridge/src/main/java/com/livefront/bridge/wrapper/BitmapWrapper.java [hunk 1]
```diff
@@ -26,18 +25,25 @@ public Bitmap getBitmap() {
...
+        mBitmap = Bitmap.createBitmap(width, height, config);
+        mBitmap.copyPixelsFromBuffer(ByteBuffer.wrap(bytes));
     }
 
     @Override
     public void writeToParcel(Parcel dest, int flags) {
-        ByteArrayOutputStream stream = new ByteArrayOutputStream();
-        mBitmap.compress(Bitmap.CompressFormat.PNG, 100, stream);
-        dest.writeByteArray(stream.toByteArray());
+        int size = mBitmap.getRowBytes() * mBitmap.getHeight();
```
Comment (weisers) [inline bridge/src/main/java/com/livefront/bridge/wrapper/BitmapWrapper.java]: `getRowBytes` has the following warning:

```
* <p>As of {@link android.os.Build.VERSION_CODES#KITKAT}, this method
* should not be used to calculate the memory usage of the bitmap. Instead,
* see {@link #getAllocationByteCount()}.
```
Are you able to use `getAllocationByteCount` instead?

Comment (byencho) [reply bridge/src/main/java/com/livefront/bridge/wrapper/BitmapWrapper.java]: Ah good call. I can double check that, yeah.

Comment (byencho) [reply bridge/src/main/java/com/livefront/bridge/wrapper/BitmapWrapper.java]: Thanks for pointing this out. So I added a version check and now use either `getAllocationByteCount` or `getByteCount` depending on the version.

Comment (weisers) [reply bridge/src/main/java/com/livefront/bridge/wrapper/BitmapWrapper.java]: Cool.  What's the minimum SDK for the library at this point?

**Review by weisers — APPROVED** [review]

File: bridge/src/main/java/com/livefront/bridge/wrapper/BitmapWrapper.java (hunk 1 above)
Comment (byencho) [reply bridge/src/main/java/com/livefront/bridge/wrapper/BitmapWrapper.java]: 14. So pretty low. And before that I believe it was 1, but I needed to add something that required API 14.

**Comment by byencho** [comment]
Thanks @weisers

**Pull request merged**